cmd/vocli/commands/account.go

```diff
@@ -13,7 +13,7 @@ var accInfoCmd = &cobra.Command{
 	Short: "Get information about an account from the vochain.",
 	Args:  cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		c, err := client.New(gatewayRpc)
+		c, err := client.New(v.GetString("url"))
```

Comment:
Given that the string `url` is repeated across multiple files under the `command` pkg maybe we can put it as a const and use this const everywhere. That way we do not need to change this string on multiple files